frameworks/cassandra/RELEASE_NOTES.md

```diff
@@ -0,0 +1,50 @@
+### Package Upgrade in Strict Mode
+
+If you are upgrading a service running in `strict` mode to the new package version (this version), you need to specify service account details as options during the package update process.
+
+
+You need to give service account details in options in order to authenticate and run in DC/OS `strict` mode. Or you set them in the GUI. Here is how a service is installed using a previous package version:
+
+```
+ $ docs package install  cassandra --options=options.json ` --package-version=<old_package_version_number>
+
+ $ cat options.json
+  {
+        "service": {
+            "principal": "this_is_your_service_account_id",
+            "secret_name": "this_is_your_sa_secret_path"
+        }
+    }
+```
+
```

Comment:
To provide your service account details, add the following parameter to a JSON options file.

**Note:** The syntax for specifying service account details has changed from the previous version. The `principal` and `secret_name` parameters have changed to `service_account` and `service_account_secret`. If you are upgrading your service from the previous version, you will need to modify your JSON options file.